samples/HttpServer_Plugins/app/application.cpp

```diff
@@ -50,8 +52,8 @@ void startWebServer()
 	 *
 	 * make sure to replace the IP address with the IP address of your HttpServer
 	*/
-	server.paths.set("/ip-n-auth", echoContentBody,
-					 new ResourceIpAuth(IpAddress("192.168.13.0"), IpAddress("255.255.255.0")), pluginBasicAuth);
+	server->paths.set("/ip-n-auth", echoContentBody,
```

Comment:
Here the Basic Auth plugins seems to be ignored. You can reproduce it by calling:
```
curl -vvv http://username:@192.168.13.10/ip-n-auth -d "1234" -H "Content-Encoding: test"
```

The request should be allowed only when the username and password match. In comparison this curl request should work:
```
curl -vvv http://username:password@192.168.13.10/ip-n-auth -d "1234" -H "Content-Encoding: test"
```

Reply:
Bug in `LinkedObject::insertAfter()`, fixed.